components/org.wso2.carbon.identity.oauth.dcr/src/test/java/org/wso2/carbon/identity/oauth/dcr/factory/UnregistrationRequestFactoryTest.java

```diff
@@ -84,19 +86,40 @@ public void testCanHandle(String requestURI, String httpMethod, boolean expected
                 "Redirect Uri doesn't match");
     }
 
-    @Test
-    public void testCreate() throws Exception {
+    @DataProvider(name = "instanceDataProvider")
+    public Object[][] getInstanceData() {
 
-        String dummyApplicationName = "dummyApplicationName";
-        String dummyUserId = "dummyUserId";
-        String dummyConsumerKey = "dummyConsumerKey";
+        unregisterRequestBuilder = mock(UnregistrationRequest.DCRUnregisterRequestBuilder.class);
+        mockHttpResponse = mock(HttpServletResponse.class);
```

Comment:
You can mock these two objects in the test method itself. Since the same object is passed, no need to pass it via a dataprovider